features/step_definitions/project_version_steps.rb

```diff
@@ -0,0 +1,16 @@
+Given /^project version (.+) exists for project (.+)(?: with creation date set (?:(1 day|(?:[2-9]|\d{2,}) days) ago))$/ do |name,project_name,nb_days|
```

Comment:
You should also add a final `?` after the last closing parenthesis, otherwise the `if nb_days` at line 7 is not useful.